### DISCUSSION TITLE
MDX support?

### DISCUSSION THREAD

**Discussion by matthewmueller** [yuin/goldmark#447] [General]
[MDX](https://mdxjs.com/) allows you to embed Components within your Markdown. 

It seems like it's implemented as a markdown plugin: https://github.com/mdx-js/mdx/tree/main/packages/remark-mdx#what-is-this

Would be great to get MDX support going for Goldmark!

**Comment by chrisbward** [comment]
Also keen! (followed you on twitter btw)
